src/components/Button/Button.tsx

```diff
@@ -16,10 +16,21 @@ export type ButtonProps = StyledProps<any> & {
 	loading?: boolean;
 	/** If the button should not display text */
 	hideText?: boolean;
+	/** All buttons must have contents */
+	children: React.ReactNode | React.ReactNodeArray;
+	/** Use these if the button should be an anchor or router link */
```

Comment:
I could not figure out how to type this in a better way sadly.

Reply:
lgtm